src/scripts/mutual_checker.json

```
@@ -6,5 +6,12 @@
     "color": "white",
     "background_color": "#777777"
   },
-  "help": "https://github.com/AprilSylph/XKit-Rewritten/wiki/Features#mutual-checker"
+  "help": "https://github.com/AprilSylph/XKit-Rewritten/wiki/Features#mutual-checker",
+  "preferences": {
+    "showOnlyMutuals": {
+      "type": "checkbox",
+      "label": "Only show posts from mutuals on dashboard",
```

```suggestion
      "label": "Only show posts from mutuals on the dashboard",
```

Minor phrasing suggestion!

---

gonna commit this one myself, i should have looked more carefully at my original suggestion

---

Thanks! I too should have checked it better. Since English is my second language, on first inspection it sounded fine.